src/messages/layouts/MessageLayout.cpp

```diff
@@ -443,12 +443,31 @@ void MessageLayout::deleteCache()
 // returns nullptr if none was found
 
 // fourtf: this should return a MessageLayoutItem
-const MessageLayoutElement *MessageLayout::getElementAt(QPoint point)
+const MessageLayoutElement *MessageLayout::getElementAt(QPoint point) const
 {
     // go through all words and return the first one that contains the point.
     return this->container_.getElementAt(point);
 }
 
+std::pair<int, int> MessageLayout::getWordBounds(QPoint relativePos) const
+{
+    const auto *element = this->getElementAt(relativePos);
+    assert(element != nullptr);
```

Comment:
getElementAt can explicitly return a nullptr if no element is matched at the position
I see the assert might be ok since we call getElementAt right before this, but since we're just now making `getWordBounds`, couldn't it accept a `MessageLayoutElement` already so the `getElementAt` call isn't done twice?

Reply:
I did it that way first, but we need `relativePos` later when calling `this->getSelectionIndex(relativePos)`. Don´t know if there is a better way to do this here (unless you want to pass in both parameters)

Reply:
Done